Graph/Topological-Sort.cpp

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+  using namespace std;
+
+int n , m; // For number of Vertices (V) and number of edges (E)
+vector< vector<int> > G;
+vector<bool> visited;
+vector<int> ans;
+
+void dfs(int v) {
+  visited[v] = true;
+  for (int u : G[v]) {
+    if (!visited[u])
+      dfs(u);
+  }
+  ans.push_back(v);
+}
+ 
+void topological_sort() {
+  visited.assign(n, false);
+  ans.clear();
+  for (int i = 0; i < n; ++i) {
+    if (!visited[i])
+      dfs(i);
+  }
+  reverse(ans.begin(), ans.end());
+}
+int main(){
+  cin >> n >> m;
```

Comment:
please add ```cout<<"enter number of vertices and edges = ";``` or something similar so the user knows what input has to be given.

Reply:
Fixed !